src/controllers/auth-controller.js

```diff
@@ -0,0 +1,112 @@
+const authService = require('../services/auth.service');
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt')
+
+module.exports = {
+
+    async register(req, res){
+        let info = req.body
+        let password = Math.random().toString(36).slice(-8)
+
+        authService.registerUser(info, password)
+            .then(user => { 
+                if(user){
+                    console.log("email " + user.email + "pass " + password);
+                    // this.sendEmail(info.email, password)
```

Comment:
Here you have commented out the sendEmail() function, and no where in your code you send the email, why have you done that? 
does this part need some further investigation?